CrashReporter/mainwindow.ui

```diff
@@ -37,95 +37,59 @@
-         </property>
-        </widget>
-       </item>
-       <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
-        <widget class="QPushButton" name="restart">
-         <property name="minimumSize">
-          <size>
-           <width>250</width>
-           <height>0</height>
-          </size>
-         </property>
-         <property name="text">
-          <string>Restart</string>
-         </property>
-        </widget>
-       </item>
-       <item row="3" column="0" colspan="2">
-        <widget class="QLabel" name="label_5">
-         <property name="font">
-          <font>
-           <pointsize>10</pointsize>
-          </font>
-         </property>
-         <property name="text">
-          <string>The tool would also benefit if you share your email and details about the crash with us.</string>
-         </property>
-         <property name="wordWrap">
-          <bool>true</bool>
-         </property>
-        </widget>
-       </item>
-       <item row="7" column="0" rowspan="2" colspan="2">
-        <widget class="QPlainTextEdit" name="plainTextEdit"/>
-       </item>
-       <item row="6" column="0">
-        <widget class="QLabel" name="label">
-         <property name="font">
-          <font>
-           <pointsize>10</pointsize>
-           <weight>75</weight>
-           <bold>true</bold>
-          </font>
-         </property>
-         <property name="text">
-          <string>Description of the crash (Optional)</string>
-         </property>
-        </widget>
-       </item>
-       <item row="9" column="1">
-        <widget class="QPushButton" name="cancel">
-         <property name="text">
-          <string>Cancel</string>
-         </property>
-        </widget>
-       </item>
-      </layout>
-      <zorder>plainTextEdit</zorder>
-      <zorder>label_4</zorder>
-      <zorder>label_5</zorder>
-      <zorder>label</zorder>
-      <zorder>reportRestart</zorder>
-      <zorder>cancel</zorder>
-      <zorder>restart</zorder>
+      <widget class="QLabel" name="label_4">
+       <property name="geometry">
+        <rect>
+         <x>12</x>
+         <y>33</y>
+         <width>521</width>
+         <height>141</height>
+        </rect>
+       </property>
+       <property name="font">
+        <font>
+         <pointsize>10</pointsize>
+         <weight>50</weight>
+         <bold>false</bold>
+         <kerning>true</kerning>
+        </font>
+       </property>
+       <property name="text">
+        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Unfortunately, &lt;span style=&quot; font-weight:600;&quot;&gt;El Maven&lt;/span&gt; has &lt;span style=&quot; font-weight:600;&quot;&gt;crashed&lt;/span&gt;. Would you like to share the report with us ?&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;NOTE&lt;/span&gt;&lt;/p&gt;&lt;p&gt;We do not store any personal identifiable information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
@rish9511 The language needs some work, I think. We can use this for reference. https://www.codeproject.com/KB/debug/crash_report/screen1.png